components/usergroup/joinAsPresenter.tsx

```diff
@@ -0,0 +1,57 @@
+import { Template } from "tinacms";
+import { tinaField } from "tinacms/dist/react";
+
+const Avatar = ({ img }) => {
+  return (
+    <div className="w-56">
+      <svg viewBox="0 0 220 250" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path fill="#E9E9E9" d="M220,250 v-105 A110,105 0 1,0 110,250 Z" />
+        <defs>
+          <clipPath id="shape">
+            <path d="M220,250 V0 H0 V145 A110,105 0 0,0 110,250 Z" />
+          </clipPath>
+        </defs>
+        {img ? (
+          <image href={img} width={220} clipPath="url(#shape)"></image>

```

Comment:
Since the image is clipped with the `shape`, if there is a better solution I'd love to refactor this part.

Love next/image for its performance!

![image](https://github.com/SSWConsulting/SSW.Website/assets/113034339/25fd449c-b474-4f16-8ed9-1398e0140411)
**Figure: clipPath on left bottom cornor**